ci-operator/jobs/openstack-k8s-operators/cinder-operator/openstack-k8s-operators-cinder-operator-master-presubmits.yaml

```diff
@@ -1,7 +1,7 @@
 presubmits:
   openstack-k8s-operators/cinder-operator:
   - agent: kubernetes
-    always_run: false
+    always_run: true
```

Comment:
needs to stay as **false**

Reply:
@bshewale can you fix this? missing this detail to merge.

Reply:
Done.